draft-ietf-mls-protocol.md

```diff
@@ -1319,7 +1319,7 @@ functions as defined in {{!RFC5869}}, as well as the functions
 defined below:
```

Comment:
The line just above here references HKDF-Extract, which is no longer used anywhere. Please remove